docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -123,19 +123,19 @@ resotoworker:
 
 ### Install Helm Chart
 
-Clone the [`someengineering/resoto`](https://github.com/someengineering/resoto) repository:
+Add the resoto helm repository:
 
 ```bash
-$ git clone https://github.com/someengineering/resoto
+$ helm repo add resoto https://helm.resoto.com
 ```
 
 Next, install Resoto using Helm:
 
 ```bash
-$ helm install resoto ./resoto/kubernetes/chart --set image.tag={{latestRelease}} -f resoto-values.yaml
+$ helm install resoto resoto/resoto -f <values.yaml>
```

Comment:
```suggestion
$ helm install resoto resoto/resoto --set image.tag={{latestRelease}} -f resoto-values.yaml
```

Reply:
We already have listed image.tag as a configurable value & have highlighted it in the readme too, by default we should point it to a LTS release as most of the OSS projects do. For a user I firmly believe installing a project should be a one-touch setup.

Reply:
Production setups should be pinning a Resoto version. Users can of course set this value to `latest` or omit it, but we should not condone this practice.

Also, `{{latestRelease}}` will be the latest stable release.

Reply:
> Also, {{latestRelease}} will be the latest stable release.

Yes but user will have to find that latestRelease by themselves right. We haven't templated it anywhere or have we 🤔

Reply:
No, `{{latestRelease}}` is replaced with the tag.

Reply:
I am still not convinced with this change but I would leave it at your discretion if we expect the user to find the tag first & then replace it.

Reply:
The user is not expected to find the tag and replace it; this becomes a copy-pastable command. Did you test or verify your changes in this PR?

Reply:
I had the confusion here but looks like this {{latestRelease}} is being rendered by docusaurus & being replaced with 2.4.0 in actual render